projections/shapes/lib/shapes/client.rb

```diff
@@ -158,7 +157,7 @@ class Client < Smithy::Client::Base
     #    - `refresh_retry_token(retry_token, error_info)`
     #    - `record_success(retry_token)`
     # @option options [Boolean] :stub_responses
-    #  When true, the client will return stubbed responses instead of networking requests.
+    #  When `true``, the client will return stubbed responses instead of networking requests.
     #  By default fake responses are generated and returned. You can specify the response data
     #  to return or errors to raise by calling {Stubs#stub_responses}.
     #  @see Stubs
```

Comment:
Is there a way to fix this weird rendering? Not sure if this is easy to do
![Screenshot 2025-04-29 at 10 13 56 AM](https://github.com/user-attachments/assets/185b7038-b8d5-499a-ba7e-5f8002cb7410)